admin-dev/themes/new-theme/scss/components/layout/_main_header.scss

```diff
@@ -83,27 +87,30 @@ $header-text-color: #4e6167 !default;
NcHogy5l5qN36uOlSR/J8vZ9Vy7I66rI/XL6tl3x1EdKIOhRkbIe442VMfFryUrdJfh3WSqI8bPhPoMWeZ5pBxpc0zMY33g+i1x7axxjOT1gjx3RdyPyOjO8ZgD5bDvzETu5yNZzn0WH6Nak1+IrAYToCPWJZm9pFVaQ+RhrNvPRJ7/ojBcUD4mkq1BnDPHT6cNme+o3LXNLdbmc6PvoiKzv+2bQe4jMtIdTw/gmNTV5e9FrPyGyPqanUozNCCc6G6iqYW8KF+2NNbYlL4kXxtzL7K4sInkaRCfhgx1H0wJM7VplX4VyTfUTXOsvJtIng3inZ5VCkvDvAvhyve11GcS5SihP7H2/s6Nc3Ux/I14WzI5FaPF8jCKsSTLySJuUdS+scjPL4dGaW1IG5weVeTvYnHck9aMDMqMxWHBce3rfY2gCHqvUQVxKoovRkXksUP2WYv8vCxlgIv5wtt96g2mg3pfOcN1SrDEmHv4gXKwZL4aIQBVfDt36isZG9y8KDr1NZFs93s/j8FOp1OskRph95akWxf2mM/pijlT/xqVMBDSpSE0GM6GzJdjMe4YhsmU02KktmdNtUKnnYJRFGPMdFnKT4m5i52oIyePVOCR57tAp/H4+2mtlvYUE4vTgJGjnBr0ZJn557SIN2opszmWQaxrX5bkqcBjEJSkJKgF3yDT6TFHePauJ8SQ+Rifai9qNVqirteYFibu9SGLYtTRqiayTjntYPdEj7mHVGsh1pAajI91O9dUyy+qKKXe+8KVZBXLsDeoyDPFZCVB1ZeEkkodkPqoND1N31XZfjkUVlY4bLse239F7iaSJ/b3wbgQIVARWR9HTaVdWI6vERyrDNkVo04pJVKfebmWWU2bC70XZpqYgshBBgmZ/l2XON12XGD7173fG8RpLeoYN8jvE6/w61SpGqcF00ZsBqvYEnkMMqN6SgZVNvzpC2sxXuuKbxeDlwdURivEMPqmsGZA/Q3KQ1aTWR9PKmLNiBX1tjDrkXG7PBswNOrAgptOb45l0Om0rJTKgJp74DAICvK1r4cNIVuauQbPKdwNXW7VzdHFIL1vNaxizOFwfhhhreWov91WVxTaSWUXylT7Uw3Cws2bicMgKNSCrYmsfZY0M51+WpSKPav6LUXucjK0QX5kgQwnpBx/ByqMZyNGxFyHSAjRo+0n5qpUn5vcl0+vEKxHi/wYlLNPnJnqfj2g2Bpw1EwmDbcR5Th4USHCR3eCB0lMSHFunT0s1HiaSzEurZQ+a9R70YjjvqMgxWLKaeF9KMTfyKxG9/pddcZhEYc+wFYHpdr/s5V7g7ycwkESlshjMrQfZsXZYjnet3oqZ7jOIS4XVkqiFHNZSw3SXlxK3n/QlU1JfR8eMhhGuzCxwUI4/iQdc2QBwyIuA2o63UU7xAbhYA6RVaVJOrPIfahSSL6XDnJNyoLl0Pu0ZL6xwdeGyGMxPyLny2euPjcdO8dtNN+56aA/a0CWV2NhXAjmlQUV6cA7l0bSAOUyHyLh5v0I1xoZ8eWduxBYLwH2O5fGeU95W+K6WQ6RcPwhDWsMJKGOmxHXDmHrZvrmi4xw4nB+U0BoTI5Fm6LQqZ/EKNY67ZZzAhvy8v9gWdj9uDlIPsD2GOKPlJmPHIowwB9cdDtUa5mxaMe4Y1iXQoN5aBAOw+iN2GCn0qLsDgnuUZNszp4qqNjiZSM04KZUVUmxiy0uxF5W6O9crUyvYvfeLiC4cPRWsc5tHZXnVBC9715MBP2n7bnmSe7SWRBB3B0uBNOzMbylIwtQsYFL2pe4pdZIwPEbQB5Xid9e3yOmp35onvuYa4NVjJcLK5er/R0FLkzTt8S1MlI1KI92z7Y5kudpuxk4JCA6trDxPyzHEqviFV7KTzqEdHBz6CJ2MNWQHxDkXESbClFIjEX/yT/z/SrthJ07Mt8YpSj8lcswmzNcJyvacBsi65hPKkwJ07A+OP5wgNJX/EZ96nQgfR2dkTs17Ut4NxaFk9CfBPHLRa1/pwfVgp8pFBNnPMeq9JTIUUYVkS+X1WjGKNk9K4TpoGLZSCN8e2wkdGG7E9P4GiyA407SmYKzHuuP2Sctg+k1EZzNLO7ZVfr5hUshiqki8km7lNPnN+gJxVM/5AXSvh+zWRUUZ3zSilHP6rOYz9f4CcPj0ixCrNwTusWtAXFoJsK+5w3CdNni13RQlOE7cI35YcEOOuYrgGORDtp0fr9BsCRj/lhZgZZrv+B5mFYbyiNlGMSZKng8O9omxdhgF8lqhPNCW9+eJINf23VZLErbleY4KkyEGxkS4vgQkKP+SDc9u546KzcfNWZ+N4nlJPsgHffNlZ89163dPOzc4e+YbInrZwnX2SvjzE0bQtfHReSe2HAdg0ycrI+xgzhsGed2cb5GtWDnGCk/rxY6Dsvxn4gd4nhnrLEXB9jqKdRzWM9XJU0PGXS1VlblLebhauRK9CScvGLUF8l8DEoobnuWdtIpp1uiANZYDmbR5WGgAmE7VN9UXvbuW0zH1eo0z2Zs4xrnUI5XqwGfSpiD38FiFMRvdJJWo6B7daewHBuEINZFwkXcLyd8jG8YgFrczLNduL3A5E7nb5CfYjs8y8TyebJKS5bRb6EYT91qFLSR5LJi2r2fiylFRZQi44TfYDjUdBo9Clp3EUnnl0HJYjwWhXf4FDryyXnq08anV6sCN110+V0sRiHFajQoEN3q1243G+IDa8+cfFXIaULMavTYcJYx0+knVJGNUZAlbYPNjroExijI1nCRQeP9qoBzQWNIuE5D5JvyQaRR2Uie2N/bA1+lYTM+rQovrZQG02LZjO3pOqrAxdKRfb8GL8MgLIL8ZBeNKIiHzI1u8LvpTF0bHOcrRtAJexqFTh3XktTdIknk3u6SaZ+nDUsR2YgLYOgBzH9nyjN1mU/snT50jl/yMehvl1brapDnY2ENkSdb2/8//XDtBvaVetwAAAAASUVORK5CYII=") left center no-repeat;
```

Comment:
This was done a long time ago with the intent of simply editing the CSS file, but I've gone a bit further since then by modifying a few template, images... So, I'm going to refactor this to use a real SVG instead of base64.